app/views/hyrax/dashboard/sidebar/_menu_partials.html.erb

```diff
@@ -0,0 +1,4 @@
+<% sidebar_partials = { activity: [], configuration: [], repository_content: [], tasks: [] } %>
```

Comment:
Is overriding this file necessary?  I wouldn't expect to need it since it should be coming from upstream Hyrax.

Reply:
Yeah it was really annoying and why it took so long `<%= render 'hyrax/dashboard/sidebar/menu_partials', menu: menu, section: :repository_content %>` Kept telling me it couldn't find this partial

Reply:
Huh, that's weird.  We should make sure to come back and fix this at some point.